src/main/scala/nl.knaw.dans.easy.sword2/DepositHandler.scala

```diff
@@ -161,9 +168,8 @@ object DepositHandler {
   }
 
   private def getBagDir(depositDir: File): Try[File] = Try {
-    val depositFiles = depositDir.listFiles.filter(isPartOfDeposit)
-    if (depositFiles.length != 1) throw InvalidDepositException(depositDir.getName, s"A deposit package must contain exactly one top-level file, which must be a directory, number found: ${depositFiles.size}")
-    if (depositFiles(0).isFile) throw InvalidDepositException(depositDir.getName, s"Deposit package contained one top-level file, but it was not a directory")
```

Comment:
Right, we now just ignore any regular files, because it is no longer prohibited for them to be in the deposit.